### DISCUSSION TITLE
Order by id desc on the first load

### DISCUSSION THREAD

**Discussion by Xatta-Trone** [rappasoft/laravel-livewire-tables#953] [Q&A]
I am trying to get the lastest results on the first page. Is there any method available that can order by id desc on the first request?

**Comment by LWlook** [comment]
How about the default sort method?

```php
$this->setDefaultSort('id', 'desc');
```

[https://rappasoft.com/docs/laravel-livewire-tables/v2/sorting/available-methods#content-setdefaultsort](https://rappasoft.com/docs/laravel-livewire-tables/v2/sorting/available-methods#content-setdefaultsort)
